index.html

```diff
@@ -0,0 +1,132 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="Product Landing Page">
+    <title>Product Landing Page for Credit and Saving company</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body id="body">
```

Comment:
Please delete the id of the body tag. The body is a unique tag, so it is unnecessary to put it an id.